c7n/resources/route53.py

```diff
@@ -95,6 +97,93 @@ def get_arns(self, resource_set):
             arns.append(self.generate_arn(_id))
         return arns
 
+@HostedZone.action_registry.register('enable-query-logging')
+class EnableQueryLogging(BaseAction):
+
+    permissions = ('route53:GetQueryLoggingConfig','route53:GetHostedZone','route53:CreateQueryLoggingConfig',
+        'route53:DeleteQueryLoggingConfig','logs:CreateLogGroup','logs:DescribeLogGroups','logs:PutRetentionPolicy')
+
+    schema = type_schema('enable-query-logging', state={'type': 'boolean'}, logretentiondays={'type': 'number'})
```

Comment:
Any thoughts on using the hyphen naming scheme here for logretentiondays in the type_schema?

Reply:
hyphens cause a "can't assign to operator" error.  Think I should use underscores?

Reply:
I might be wrong, but I think it just needs to be 

```
schema = type_schema('enable-query-logging', state={'type': 'boolean'}, 'log-retention-days'={'type': 'number'})
```

Reply:
Hyphens will not work here though:
logretentiondays = self.data.get('log-retention-days', 30)

I was thinking to keep the var the same.

Reply:
Ah I see, yeah let's keep it the same